daemon/containerd/image_pull.go

```diff
@@ -90,3 +125,15 @@ func (i *ImageService) PullImage(ctx context.Context, image, tagOrDigest string,
 	}
 	return nil
 }
+
+// writeStatus writes a status message to out. If layersDownloaded is true, the
+// status message indicates that a newer image was downloaded. Otherwise, it
+// indicates that the image is up to date. requestedTag is the tag the message
+// will refer to.
+func writeStatus(out progress.Output, requestedTag string, layersDownloaded bool) {
+	if layersDownloaded {
```

Comment:
Wouldn't it be easier and more correct to just check if the image `Target` changed after pull?

It would also display the correct message if anything "above" manifest changed (manifest list).

Reply:
Done